test/lib/props.test.ts

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node_modules/.bin/ts-node
+// Shebang is required, and file *has* to be executable: chmod +x file.test.js
+// See: https://github.com/tapjs/node-tap/issues/313#issuecomment-250067741
+// tslint:disable:max-line-length
+// tslint:disable:object-literal-key-quotes
+import { test } from 'tap';
+import * as fs from 'fs';
+import { extractProps } from '../../lib';
+import { InvalidUserInputError } from '../../lib/errors';
+
+test('.props file is parsed', async (t) => {
+  const propsFileContents = fs.readFileSync(`${__dirname}/../fixtures/dotnet-with-props/Packages.props`, 'utf-8');
+  const result = await extractProps(propsFileContents);
+  // tslint:disable:object-literal-sort-keys
```

Comment:
just sort keys, instead of doing this, please. Or let's disable `object-literal-sort-keys` linter rule. Cos it's not useful at all